R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2070,6 +2072,16 @@ test_that("dapply() on a DataFrame", {
   rownames(expected) <- NULL
   expect_identical(expected, result)
 
+  result <- dapplyCollect(
+              df,
+              function(x) {
+                y <- x[x$a > 1, ]
+                y <- cbind(y, y$a + 1L)
```

Comment:
I see. In our previous product we could do that. Pass other parameters. I'll take a look and might have some suggestions.
But I think this is not specifically related to this pull request.
LGTM!